commands/builds/create.js

```diff
@@ -9,11 +9,18 @@ let request = require('request')
 let exec = require('child_process').execSync
 let nodeTar = require('../../lib/node_tar')
 
-function compressSource (tar, cwd, tempFile, cb) {
+function compressSource (context, tar, cwd, tempFile, cb) {
   let tarVersion = exec(tar + ' --version').toString()
 
   if (tarVersion.match(/GNU tar/)) {
-    exec(tar + ' cz -C ' + cwd + ' --exclude .git --exclude .gitmodules . --exclude-vcs-ignores > ' + tempFile)
+    let excludeVcsIgnore = context.flags['exclude-vcs-ignore'] || 'true'
+    let command = tar + ' cz -C ' + cwd + ' --exclude .git --exclude .gitmodules .'
+
+    if (excludeVcsIgnore === 'true') {
```

Comment:
it's a little weird to have a hasValue flag have a boolean value. I think it would be better to make the flag a boolean and call it `--include-vs-ignores`